### ISSUE TITLE
About the details of model ablations

### ISSUE THREAD

**Issue by lairikeqiA** [google-research/tapas#85]
In the model ablations session,  what is the mean of '-table pre-training' in table 6 on WTQ dataset? Does '-table pre-training' mean the model is initialized randomly? And the result is based on the large model in table 6?

**Comment by thomasmueller-google** [comment]
IIRC, it means that the model is initialized from a normal Bert checkpoint.
That is no Mask-LM training on tables was performed.

Yes, all models in that paper are large models.

**Comment by lairikeqiA** [comment]
Does '-table pre-training'  mean that it is trained on the training dataset of WTQ directly and test on the test dataset of WTQ?

**Comment by thomasmueller-google** [comment]
Yes, but you start with a normal BERT checkpoint, so most parameters have been pre-trained.
Only the new parameters (row / column embeddings, output layers, ...) are randomly initialized.

**Comment by eisenjulian** [comment]
Will go ahead and close this issue, feel free to re-open or create a new issue for new questions.
